core/templates/pages/contributor-dashboard-admin-page/translation-role-editor-modal/cd-admin-translation-role-editor-modal.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="modal-header e2e-test-language-selector-modal">
+  <h4 class="modal-title">Translation reviewer roles for user {{username}}</h4>
+</div>
+<div class="modal-body">
+  <div class="modal-body">
+    <p *ngIf="assignedLanguageIds.length">User has review rights in the following languages:</p>
+    <p *ngIf="!assignedLanguageIds.length">Please assign user to atleast one language:</p>
+    <ul>
+      <li *ngFor="let languageId of assignedLanguageIds">
+        <div class="d-flex justify-content-between">
+          <span>{{ languageIdToName[languageId] }}</span>
+          <div class="mat-icon-button oppia-progress-spinner" *ngIf="languageIdInUpdate === languageId">
+            <mat-spinner [diameter]="24"></mat-spinner>
+          </div>
+          <button mat-button matSuffix mat-icon-button
+                  *ngIf="languageIdInUpdate !== languageId"
+                  aria-label="Clear"
+                  (click)="removeLanguageId(languageId)">
+            <mat-icon matListIcon>close</mat-icon>
+          </button>
+        </div>
+      </li>
+    </ul>
+    <div>Add Language:</div>
+    <div *ngIf="languageIdsForSelection.length">
+      <div class="d-inline-flex mt-3 w-100 language-dropdown">
+        <select class="w-50 e2e-test-language-selector" placeholder="Select new language to add" label="Language selector" ngbAutofocus [(ngModel)]="newLanguageId">
+          <option *ngFor="let languageId of languageIdsForSelection" [value]="languageId">{{ languageIdToName[languageId] }}</option>
+        </select>
+        <button mat-icon-button color="primary" (click)="addLanguage()" class="e2e-test-language-selector-add-button">
+          <mat-icon aria-label="Example icon-button with a heart icon">add</mat-icon>
+        </button>
+      </div>
+    </div>
+    <div *ngIf="!languageIdsForSelection.length">
+      <i *ngIf="assignedLanguageIds.length">There are no languages remaining, looks like user is assigned to all the languages.</i>
+    </div>
+  </div>
+</div>
+<div class="modal-footer">
+  <button type="button" class="cancel-button" (click)="close()"> save and Close</button>
```

Comment:
Should the text just be "Close"? Clicking the button doesn't seem to "save" anything. This might be confusing if the user is expecting their changes to be saved only after clicking the button.

Reply:
Sg. Updated.